apps/testing/src/main/java/com/akto/testing/Main.java

```diff
@@ -130,6 +130,8 @@ private static TestingRun findPendingTestingRun() {
     }
 
     private static TestingRunResultSummary findPendingTestingRunResultSummary() {
+
+        // if you change this delta, update this delta in method getCurrentRunningTestsSummaries
         int now = Context.now();
         int delta = now - 20*60;
```

Comment:
make the delta as a final static variable, and use it in all places where it might be required to be changed. This will ensure single source of truth.